ibm/flex/structures.go

```diff
@@ -1944,6 +1945,16 @@ func GetLocation(instance models.ServiceInstanceV2) string {
 		return cName + "-" + region
 	}
 }
+func GetLocationV2(instance rc.ResourceInstance) string {
+	crnSplit := strings.Split(*instance.CRN, ":")
```

Comment:
Instead of spliting this way where we have chance of nil pointer or index range errors. Can you use below package 
https://github.com/IBM-Cloud/bluemix-go/blob/4d6050d2dbc5e6d08e91e7a788438fce0ff9c681/crn/crn.go#L82